src/main/kotlin/blackjack/view/OutputView.kt

```diff
@@ -13,68 +13,81 @@ object OutputView {
         return "${face}${symbol}"
     }
 
-    fun showHandCards(participant: Participant, isFirstRound: Boolean): String {
-        return when {
-            participant is Dealer && isFirstRound -> "${participant.name}: ${showCardFaceAndSymbol(participant.cardsInHand.cards[0])}"
+    fun showDealerFirstHandCards(dealer: Dealer): String {
+        return ("Dealer: ${showCardFaceAndSymbol(dealer.cardsInHand.cards[0])}")
+    }
```

Comment:
Finally! Now we can focus on the game without even knowing the dealer’s name!
And if we ever want to change how the dealer appears, we only need to look at the view 👍